x-pack/plugin/eql/src/test/resources/queryfolder_tests.txt

```diff
@@ -145,6 +145,49 @@ process where endsWith(user_name, "c")
 {"wildcard":{"user_name":{"wildcard":"*c","boost":1.0}}}],"boost":1.0}}
 ;
 
+fieldNoEquals
+process where bool
+;
+{"bool":{"must":[
+{"term":{"bool":{"value":true,
+;
+
+fieldNoEqualsInExpression
+process where length(file_name) > 0 and bool
+;
+{"bool":{"must":[
```

Comment:
Can you add more from the actual query DSL. The fact that there is a `bool` `must` and a `term` with `true` doesn't mean the actual query is entirely correct, more like how we do it with SQL (where the entire query is checked for correctness). For `term` at least, there is the new parameter that Mark H. added that makes it case insensitive or not.

Reply:
I'm not a fan of comprehensive query tests since any optimization or change breaks the test. For example the bool must is separate since the term comparison gets applied on process as well.
The aim of this test is to check that the bool field is compared not to what. So checking the parameter for case insensitivity (which btw, does appear only if it gets set) goes beyond the scope tested here.